pwiz_tools/Skyline/TestFunctional/DetectionsPlotTest.cs

```diff
@@ -174,29 +179,29 @@ protected override void DoTest()
         private void AssertDataCorrect(DetectionsPlotPane pane, int refIndex, float qValue, bool record = false)
         {
             DetectionPlotData data = null;
-            WaitForConditionUI(() => (data = pane.CurrentData) != null 
-                                           && data.QValueCutoff == qValue
+            WaitForConditionUI(() => pane.CurrentData != null 

```

Comment:
Lots of unnecessary white space changes above. Is this the critical change? Using CurrentData everywhere instead of assigning it in the first logical expression and then using the assigned value?